samples/csharp_dotnetcore/52.teams-messaging-extensions-search-auth-config/Bots/TeamsMessagingExtensionsSearchAuthConfigBot.cs

```diff
@@ -8,6 +8,7 @@
 using System.Net.Http;
 using System.Threading;
 using System.Threading.Tasks;
+using AdaptiveCards;
```

Comment:
If we are importing AdaptiveCards library, and not using the CreateAdaptiveCardAttachment to load from Resources, then we should remove the method: CreateAdaptiveCardAttachment as well as the adaptiveCard.json file.

Reply:
Removed unused method along with adaptive card json file.